PowerEditor/src/Parameters.h

```diff
@@ -1672,6 +1672,16 @@ class NppParameters final
 		_cmdSettingsDir = settingsDir;
 	};
 
+	void setTitleBarAdd(const generic_string& titleAdd)
+	{
+		_titleBarAdditional = titleAdd;
+	}
+
+	generic_string getTitleBarAdd()

```

Comment:
It's better to declare as `const generic_string& getTitleBarAdd() const {...}`

Reply:
True.  Something felt strange while I was doing it.  :)

Reply:
Changed.  Thanks @mere-human.